src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs

```diff
@@ -194,6 +187,10 @@ private static LoggerConfiguration ConfigureConsoleSink(this LoggerConfiguration
         /// <param name="config">The configuration for the appender.</param>
         private static LoggerConfiguration ConfigureFileSink(this LoggerConfiguration loggerConfiguration, ILogConfig config)
         {
+            if (!config.Enabled)
```

Comment:
Do we need to add similar code in `ConfigureAuditLogSink()`?

Reply:
I meant to ask the team this but forgot about it. Is logging disabled but audit log enabled a valid configuration? What should happen?

Reply:
Seems to me that if logging is disabled (because of read-only FS), audit logging has to be disabled as well. But as long as the app doesn't crash when the audit log sink tries to create the log file, I guess it isn't critical.

Reply:
I've explicitly disabled it now if you have logging disabled; I think that makes sense.

Reply:
That seems like a scenario that it would be great to warn a customer about, but unfortunately with logging disabled there is no way to provide a warning about it.

Reply:
True, we'll have to rely on the supportability metric.